client/src/utils/axiosGet.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+const checkResponse = response => {
+    if (response.status !== 200) {
+      console.log(`Error with the request! ${response.status}`);
+      return;
+    }
+    return response;
+  };
+const  axiosGet = url =>{
+
+    return axios.get(url)
+    .then(checkResponse)
+    .catch(error => {throw new Error(`Could not get data , error:${error}`)})
```

Comment:
throw will crash the website, so you need to make sure where ever you are using this function that it's wrapped around `try {} catch() {}`